kanidm_rlm_python/Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM opensuse/tumbleweed:latest
 
 EXPOSE 1812 1813
 
+
 RUN zypper --gpg-auto-import-keys refresh --force
```

Comment:
Can you add "zypper ar -f obs://home:firstyear:branches:network home:firstyear:branches:network " here before you install the free radius server?

Reply:
```
7ef3a792a088:/etc/zypp/repos.d # ldd /usr/lib64/freeradius/rlm_python3.so
	linux-vdso.so.1 (0x00007ffd77fdf000)
	libpython3.10.so.1.0 => /lib64/libpython3.10.so.1.0 (0x00007fda4c262000)
	libc.so.6 => /lib64/libc.so.6 (0x00007fda4c033000)
	/lib64/ld-linux-x86-64.so.2 (0x00007fda4c5c2000)
	libm.so.6 => /lib64/libm.so.6 (0x00007fda4bf4b000)
```

looks resolved in a test, submitted to opensuse